prow/plugins.yaml

```diff
@@ -2,50 +2,63 @@
 # Keys: Full repo name: "org/repo".
 # Values: List of plugins to run against the repo.
 ---
-google/cadvisor:
```

Comment:
Can you get a commit that moves from the old to the new schema and a follow-up with the new config?